tests/test_export_framework_results_reasons.py

```diff
@@ -0,0 +1,370 @@
+from unicodecsv import reader
+
+import pytest
+from mock import Mock, patch
+
+from dmscripts.export_framework_results_reasons import export_suppliers
+
+from .assessment_helpers import BaseAssessmentTest, BaseAssessmentOnFrameworksAsThoughNoBaselineTestMixin
+
+
+class _BaseExportFrameworkResultsReasonsTest(BaseAssessmentTest):
+    def _mock_get_questions_numbers_from_framework_impl(self, framework_slug, content_loader):
+        assert content_loader is self.mock_content_loader
```

Comment:
Checks the code under test is sending the args in the right order. Yes, I did screw this up during development.

It's also a symmetry thing, routinely check all args are what you expect them to be - the "pre-canned" reply the code should expect *given* these specific args.